templates/poster.html

```diff
@@ -1,25 +1,20 @@
-{% set page_title = paper.content.title %}
-{% extends "base.html" %}
-
-{% block head %}
-
-{{ super() }}
+{% set page_title = paper.content.title %} {% extends "base.html" %} {% block
+head %} {{ super() }}
 <meta name="citation_title" content="{{paper.content.title}}" />
```

Comment:
Why did this change?

Reply:
I saw iclr repo shows the session times on the poster site too and I liked that implementation. I thought it will be great if we add times over there too.

Reply:
Not sure I understand. It seems like these changes are mostly white space?

Reply:
Oh, you are right. I removed those now.